v-next/hardhat/test/internal/builtin-plugins/solidity/config.ts

```diff
@@ -482,11 +483,11 @@ describe("solidity plugin config resolution", () => {
 
       assert.equal(
         resolvedConfig.solidity.profiles.production.preferWasm,
-        true,
+        true && shouldUseWasm(),
```

Comment:
```suggestion
        shouldUseWasm(),
```
I can't think of a gain here from `true && shouldUseWasm()`